src/test/java/com/ericsson/bss/cassandra/ecaudit/auth/TestWhitelistOptionParser.java

```diff
@@ -113,4 +135,17 @@ public void testParseInvalidResource()
         assertThatExceptionOfType(InvalidRequestException.class)
         .isThrownBy(() -> parser.parseResource("guck/ks/tbl1"));
     }
+
+    @Test
+    public void testParseDropLegacyValue()
+    {
+        parser.parseDropValue("NOW");
```

Comment:
I see no assertion here, what is this testing?

Reply:
Hmm, it's verifying that an exception is _not_ thrown.

Would it be more clear to rename method to verifyDropValue?